frontend/src/components/feed/Feed.cy.js

```diff
@@ -29,9 +29,9 @@ describe("Feed", () => {
   it("sends a post request to /posts and response a OK message", () => {
     window.localStorage.setItem("token", "fakeToken");
 
-    cy.intercept("POST", "/posts", { message: "OK" }).as("newPostRequest");
-    cy.mount(<Feed navigate={navigate} />);
+    cy.intercept("POST", "/posts", {message: "OK"}).as("newPostRequest")
 
+    cy.mount(<Feed navigate={navigate} userData={{firstName: "james", lastName:"Mcleish"}}/>);
```

Comment:
[nit]: Should the first letter of name be capital?

Reply:
Yes if should be, but this is just for the test!